Make component multi-instance capable (#16)
vshn/exoscale-metrics-collector: docs/modules/ROOT/pages/references/parameters.adoc

You need to get the token from the https://portal.exoscale.com[Exoscale Console].
You need to select the correct account (token is limited to one account), choose "IAM" in the menu and generate a new key pair.

> == `secrets.exoscale.stringData.EXOSCALE_API_SECRET`

Reviewer: ```suggestion
== `secrets.credentials.stringData.EXOSCALE_API_SECRET`
```

The Exoscale API secret.

Second part of the Exoscale API credentials.
See EXOSCALE_API_KEY for instructions.

> == `secrets.exoscale.stringData.K8S_SERVER_URL`

Reviewer: ```suggestion
== `secrets.credentials.stringData.K8S_SERVER_URL`
```

[horizontal]
type:: string
default:: Required.

The Kubernetes server URL.

> == `secrets.exoscale.stringData.K8S_TOKEN`

Reviewer: ```suggestion
== `secrets.credentials.stringData.K8S_TOKEN`
```

[horizontal]
type:: string
default:: Required.

The token to connect to a Kubernetes cluster.

> The Service Account connected to this token should have access to bucket managed resource.

Reviewer: "Access" is too unspecific for describing permissions.
```suggestion
The Service Account connected to this token should have `get` and `list` permissions to `buckets.exoscale.crossplane.io` managed resource.
```
